hw/ip/csrng/doc/_index.md

```diff
@@ -234,7 +235,7 @@ This section describes the application interface, which is required for performi
 Each CSRNG instance corresponds to a unique application interface port, which implements the application interface described here.
 Any hardware peripherals which require complete control of an instance may connect directly to a dedicated interface port.
 Meanwhile peripherals without any special requirements (i.e. personalization strings or non-FIPS-approved, fully-deterministic number sequences) may share access to an instance via the entropy distribution network (EDN) IP.
-The EDN's manage the instantiation and reseeding of CSRNG instances for general use-cases, providing either on-demand or timed-delivery entropy streams to hardware peripherals.
+The EDNs manage the instantiation and reseeding of CSRNG instances for general use-cases, providing either on-demand or timed-delivery entropy streams to hardware peripherals.
```

Comment:
similar to `entropy_src` above, should we change this to `edns`

Reply:
EDN and CSRNG are abbreviations that have been introduced in this document, so I think it's meaningful to keep them. In contrast, "ENTROPY_SRC" is neither a module name nor an abbreviation nor the identifier used in the code, so I replaced it.